io.github.jeffshee.Hidamari.json

```diff
@@ -0,0 +1,240 @@
+{
+    "app-id": "io.github.jeffshee.Hidamari",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "command": "hidamari",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--filesystem=home:ro",
+        "--filesystem=xdg-config/hidamari",
+        "--filesystem=xdg-config/autostart",
+        "--own-name=io.github.jeffshee.Hidamari.player",
+        "--own-name=io.github.jeffshee.Hidamari.server",
+        "--talk-name=org.gnome.Shell",
+        "--talk-name=org.gnome.Shell.Extensions",
+        "--talk-name=org.gnome.SessionManager",
+        "--talk-name=org.gnome.ScreenSaver",
+        "--talk-name=org.cinnamon.ScreenSaver",
+        "--talk-name=org.kde.screensaver",
+        "--talk-name=org.freedesktop.ScreenSaver",
+        "--talk-name=org.kde.StatusNotifierWatcher",
+        "--talk-name=org.freedesktop.Flatpak",
+        "--system-talk-name=org.freedesktop.login1"
```

Comment:
please why you need all of these D-Bus permissions.

Reply:
@hfiguiere Hi, thanks for the review.
Hidamari is a live wallpaper application.
 `org.*.ScreenSaver` is for monitoring the event of screen lock. When a screen lock happens, the app will pause the live wallpaper playback.
 `org.kde.StatusNotifierWatcher` Apparently it is a requirement for displaying the systray icon, which is an option of the app.
 `org.freedesktop.Flatpak` is required to change the system wallpaper by calling `gsettings` on the host. Currently, this is the only option due to the limitation of Flatpak I believe.
 `org.freedesktop.login1` is for monitoring the event of the session ending (power-off, restart, etc). The app has to restore the system wallpaper to the user's original one before exiting.